src/Sake.Library/Shared/_exec.shade

```diff
@@ -10,19 +10,55 @@ log info="  program: ${program}"
 log info="  commandline: ${commandline}"
 log info="  workingdir: ${workingdir}"
 
+
+functions
+    @{
+        bool __WriteExecOutputToLogger { get; set; }
```

Comment:
This is used to signal if we want to redirect the process output and write it to the logger. This is set as part of `_use_teamcity`. Thoughts on doing this?